monai/data/meta_tensor.py

```diff
@@ -477,6 +477,10 @@ def pixdim(self):
             return [affine_to_spacing(a) for a in self.affine]
         return affine_to_spacing(self.affine)
 
+    def set_pixdim(self) -> None:
```

Comment:
Does it make sense for this to be a method? If it's only going to be called in one place it's simple code could just be put there. If there's anticipation that this would be called by other things then that's fine.